tests/cases/compiler/acceptSymbolAsWeakType.ts

```diff
@@ -0,0 +1,23 @@
+// @strict: true
+// @lib: esnext
+// @target: esnext
+
+const s: symbol = Symbol('s');
+
+const ws = new WeakSet([s]);
+ws.add(s);
+ws.has(s);
+ws.delete(s);
+
+const wm = new WeakMap([[s, false]]);
+wm.set(s, true);
+wm.has(s);
+wm.get(s);
+wm.delete(s);
+
+const wr = new WeakRef(s);
+wr.deref();
+
+const f = new FinalizationRegistry<symbol>(() => {});
+f.register(s, null);
+f.unregister(s);
```

Comment:
Small tweak here. We are getting [an error](https://github.com/bloomberg/TypeScript/blob/7563f5e7670289844fc880bbc84fe38102cd14de/tests/baselines/reference/acceptSymbolAsWeakType.errors.txt#L24) in the baseline for this input, this is because we are declaring that the `FinalizationRegistry` 'heldValue' type is `symbol` but passing `null`. It we remove the generic type annotation on the constructor then we can happily pass anything as the 2nd argument to `register`.

```suggestion
const f = new FinalizationRegistry(() => {});
f.register(s, null);
f.unregister(s);

```